src/stylesheets/screens/survey-detail.scss

```diff
@@ -0,0 +1,66 @@
+.survey-detail {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__header-left {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__arrow-left {
+    display: block;
+    width: rem(15px);
+    height: rem(15px);
+    margin-left: rem(5px);
+
+    border-top: rem(2px) solid map-get($brand-white, 'primary');
```

Comment:
@rafayet-monon This is such a small amount of unit, we shouldn't `rem()` for this.
Because when it turns to `rem` with a small amount, it will make our border not look sharp enough.

Reply:
Removed it as it won't be used anymore. From which point should we use rem?